policyengine_us/parameters/gov/states/ny/tax/income/credits/ctc/additional/min_ctc.yaml

```diff
@@ -0,0 +1,12 @@
+description: New York provides the Empire State Child Credit for filers who receive a base credit above this amount.  
+values:
+  2021-01-01: 25
+metadata:
+  unit: currency-USD
+  period: year
+  label: New York additional Empire State Child Credit minimum base credit
+  reference:
+    - title: New York Instructions for Form IT-201 Line 40
```

Comment:
this source doesn't say the $25 thing

also for other places the title doesnt match the href